tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml_iteration.bal

```diff
@@ -103,6 +100,7 @@ function chainedIterableOps() returns xml {
 xml theXml = xml `<book>the book</book>`;
 xml bitOfText = xml `bit of text\u2702\u2705`;
 xml compositeXml = theXml + bitOfText;
+string output = "";
```

Comment:
Why do we need a module level variable? Why can't we use local variables and pass them as arguments to `concatString`?